examples/offline_inference_vision_language.py

```diff
@@ -496,7 +500,10 @@ def main(args):
             },
         } for _ in range(args.num_prompts)]
 
+    start_time = time.time()
     outputs = llm.generate(inputs, sampling_params=sampling_params)
+    elapsed_time = time.time() - start_time
```

Comment:
nit

Reply:
Second this - the intention of this script is more for showing example usage.